### PR TITLE
Legal and privpolicy link

### PR DESCRIPTION
**Pull request by einhirn**
I added two items in "site settings" as suggested in #1268.
When you put an URL in there it gets added as "legal" or privacy policy link, respectively, in the page footer. If left empty, it doesn't display.

### CODE REVIEW

**Comment by Sea444** [comment]
Would be great if this PR could be merged into Master with upcoming 2.5.5. 
Could this be possible @farhatahmad ? :-)

File: config/locales/de_DE.yml [hunk 1]
```diff
@@ -43,6 +43,18 @@ de_DE:
...
+      legal:
+        change: URL ändern
+        info: Die URL für den Link zum Impressum am Seitenende ändern
+        placeholder: Impressums-URL...
+        title: Impressum
+        invalid: Ungültige URL
+      privpolicy:
+        change: URL ändern
+        info: Die URL für den Link zur Datenschutzerklärung am Seitenende ändern
+        placeholder: Datenschutzerklärungs-URL...
```
Comment (lkiesow) [inline config/locales/de_DE.yml:55]:
```suggestion
        placeholder: URL der Datenschutzerklärung…
```

File: config/locales/de_DE.yml [hunk 2]
```diff
@@ -43,6 +43,18 @@ de_DE:
         placeholder: Bild URL...
         title: Markenlogo
         invalid: Ungültige URL
+      legal:
+        change: URL ändern
+        info: Die URL für den Link zum Impressum am Seitenende ändern
+        placeholder: Impressums-URL...
```
Comment (lkiesow) [inline config/locales/de_DE.yml:49]:
```suggestion
        placeholder: URL des Impressums...
```

**Comment by farhatahmad** [comment]
@Sea444 Not in 2.5.5 as we're releasing that tomorrow (assuming nothing goes wrong). We're working to move to shorter release cycles so this will most likely get included in 2.5.6 release next week

**Comment by farhatahmad** [comment]
@einhirn Thanks for the PR. Can you take the German translations out? The PR should only have the English strings. Once it's merged, the translations can be made in Transifex

If the translations are done this way, I believe Transifex will override/delete them on next pull

**Comment by einhirn** [comment]
Ok, I removed the german translations.

**Pull request merged**